runtime/autoload/netrw.vim

```diff
@@ -702,6 +702,11 @@ fun! netrw#Explore(indx,dosplit,style,...)
   endif
 
   " save registers
+  if has("clipboard") && g:netrw_clipboard
+"   call Decho("(netrw#Explore) save @* and @+",'~'.expand("<slnum>"))
+   sil! let keepregstar = @*
+   sil! let keepregplus = @+
```

Comment:
These `let @+ = ...` calls were intentionally removed in https://github.com/neovim/neovim/commit/6c22c7ab97cca9f8dda6863ee7f1db1ce30a3451 for performance.

netrw is super noisy about writing to the clipboard, which is slow in Nvim. 

Nvim clipboard could be made faster with an approach like https://github.com/neovim/neovim/issues/11804#issuecomment-1113927971

Reply:
Ah, I see. Do you think vim would be interested in this patch?